RxJavaTemplate[DEPRECATED]/buildSrc/src/main/java/Versions.kt

```diff
@@ -2,7 +2,7 @@ object Versions {
     const val BUILD_GRADLE_VERSION = "4.2.2"
 
     const val ANDROID_VERSION_CODE = 9
-    const val ANDROID_VERSION_NAME = "3.7.0"
+    const val ANDROID_VERSION_NAME = "3.8.0"
```

Comment:
Do we need to maintain the DEPRECATED template too? 🤔

Reply:
Hmmm, true. No need to 👍